WORKSPACE

```diff
@@ -113,9 +120,16 @@ new_git_repository(
     remote = "https://github.com/googleapis/googleapis.git",
 )
 
+# Dependency for org_golang_google_grpc
+new_go_repository(
```

Comment:
Manager seems to be doing the same - I believe if we don't define it explicitly an older version will be picked anyways.

IMHO having one big WORKSPACE with all the deps explicitly visible and defined - and not using bzl scripts - would
be much cleaner, and worth the effort in particular if we do it in istio repo and use it across components.